content/lua/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'Data Type' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'In Lua, there are many Data Types. Commonly used data types are: Number, Boolean, String, and Nil.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+ - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Booleans'
+  - 'Data Types'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+In Lua, there are many **Data Types**. Some commonly used data types are:
+* Number
```

Comment:
```suggestion
* Number: Represents double-percision floating point values (there are is no `int` type in Lua). 
```

Reply:
Boolean and string are fairly straightforward but I think number and nil are less common. It might be good to briefly define each of these as above

Reply:
that makes sense thanks you